Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/TargetsTests/RazorTargetTests.cs

```diff
@@ -35,6 +35,8 @@ public class RazorTargetTests
 @"<SonarQubeConfigPath>PROJECT_DIRECTORY_PATH</SonarQubeConfigPath>
   <SonarQubeTempPath>PROJECT_DIRECTORY_PATH</SonarQubeTempPath>";
 
+        private static readonly char Separator = Path.DirectorySeparatorChar;
```

Comment:
this is not really great, most likely we'd need to have a static helper that , for tests involving paths, would replace `\\` with `Path.DirectorySeparatorChar`...

or consider how to do this implicitly inside the test infrastructure somehow

for now I'd leave this as is